sierra_adapter/sierra_bib_merger/src/main/scala/uk/ac/wellcome/platform/sierra_bib_merger/modules/SierraBibMergerModule.scala

```diff
@@ -1,10 +1,35 @@
 package uk.ac.wellcome.platform.sierra_bib_merger.modules
 
+import javax.inject.Singleton
+
 import akka.actor.ActorSystem
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsync
+import com.google.inject.Provides
+import com.twitter.app.Flag
 import com.twitter.inject.{Injector, TwitterModule}
-import uk.ac.wellcome.platform.sierra_bib_merger.services.SierraBibMergerWorkerService
-
-object SierraBibMergerModule extends TwitterModule {
+import uk.ac.wellcome.metrics.MetricsSender
+import uk.ac.wellcome.models.aws.DynamoConfig
+import uk.ac.wellcome.platform.sierra_bib_merger.services.{
+  SierraBibMergerUpdaterService,
+  SierraBibMergerWorkerService
+}
+import uk.ac.wellcome.utils.TryBackoff
+
+object SierraBibMergerModule extends TwitterModule with TryBackoff {
+
+  override lazy val continuous: Boolean = false
```

Comment:
Why is this false?

Reply:
This is a TryBackoff parameter – it means that after the first success, we should continue to process messages rather than exit as succeeded.

Reply:
probably late, but _i think_ the behaviour is the opposite. If it's false it will stop after the first message

Reply:
```scala
  * Additionally, the operation can run again after it succeeds (reverting
  * to the base wait time), or give up after the first success.  This is
  * controlled by a third attribute:
  *
  *     @param continuous      true if the operation should repeat, false
  *                            if it should return after the first success
```

oops, you're right